app/services/curation_concerns/actors/actor_factory.rb

```diff
@@ -9,6 +9,7 @@ def self.build(curation_concern, current_user)
 
       def self.stack_actors(curation_concern)
         [AddToCollectionActor,
+         AddAsMemberOfCollectionsActor,
```

Comment:
I don't think Collections use the ActorStack, so I think we can get rid of it, right?